File: 백준/Silver/7568. 덩치/덩치.cc
#include <iostream>
using namespace std;

int main() {
    int N = 0;
    cin >> N;
    int lst[50][2] = {0};
    int ans[50] = {0};

    for (int i = 0; i < N ; i++){
        cin >> lst[i][0] >> lst[i][1];
        ans[i] = 1;
    }

    for (int i = 0 ; i < N ; i++){
        for (int j = 0 ; j < N ; j++){
            if (lst[i][0] < lst[j][0] && lst[i][1] < lst[j][1]) {
                ans[i] ++;
            }
        }
    }

    for (int i = 0; i < N; i++){
        cout << ans[i] << " ";
    }

    return 0;

}

File: 백준/Silver/7568. 덩치/덩치.py
N = int(input())
lst = []
for i in range(N):
    lst.append(list(map(int,input().split())))

ans = [0]*N
for i in range(len(lst)):
    a,b = lst[i][0],lst[i][1]
    cnt = 1
    for x,y in lst:
        if a < x and b < y:
            cnt+=1
    ans[i]= cnt
    
print(*ans)